Doc/Manual/Extending.html

```diff
@@ -2991,7 +2991,7 @@ <H3><a name="Extending_nn38">40.10.7 Low-level code generators</a></H3>
   /* create the wrapper object */
   Wrapper *wrapper = NewWrapper();
 
-  /* create the functions wrappered name */
+  /* create the functions wrapped name */
```

Comment:
I think that should really say "create the wrapper function's name" - I'll merge this and then push a fix for that.